invenio/b2share/modules/b2deposit/b2share_model/model.py

```diff
@@ -16,14 +16,17 @@
 ## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
 import collections
+import csv
+import os.path
 
 from invenio.ext.sqlalchemy import db
 from flask import current_app
 from datetime import date
 
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Yes, I agree it would but I didn't use it because of the reasons below:

1. http://stackoverflow.com/questions/779495/python-access-data-in-package-subdirectory#comment25649364_5601839
1. `%time import pkg_resources
CPU times: user 42.1 ms, sys: 12.8 ms, total: 54.8 ms
Wall time: 71.2 ms`
1. `%time import os.path
CPU times: user 7 µs, sys: 0 ns, total: 7 µs
Wall time: 10 µs`
1. `__package__` is broken in Python 3.0 - 3.2